data/en/structfind.json

```diff
@@ -1,13 +1,14 @@
 {
 	"name":"structFind",
 	"type":"function",
-	"syntax":"structFind(structure, key)",
+	"syntax":"structFind(structure, key [, defaultValue ])",
 	"returns":"any",
-	"related":[],
-	"description":" Determines the value associated with a key in a structure.",
+	"related":["structFindKey", "structFindValue"],
+	"description":"Determines the value associated with a key in a structure.",
 	"params": [
-		{"name":"structure","description":"Structure that contains the value to return","required":true,"default":"","type":"struct","values":[]},
-		{"name":"key","description":"Key whose value to return","required":true,"default":"","type":"string","values":[]}
+		{"name": "structure", "description": "Structure that contains the value to return", "required": true, "default": "", "type": "struct", "values": []},
+		{"name": "key", "description": "Key whose value to return", "required": true, "default": "", "type": "string", "values": []},
+		{"name": "defaultValue", "description": "Default value which will be returned if the key does not exist or if null was found. Currently only supported by Lucee. See http://docs.lucee.org/reference/functions/structfind.html#argument-defaultValue", "required": false, "default": "", "type": "any"}
```

Comment:
You can put Lucee4.5+ engine label at the beginning of the description.

Reply:
Done in https://github.com/foundeo/cfdocs/pull/869/commits/d5680f8480cc532f7f8953475d0046f40b5db0e5.